### PR TITLE
[new] add pqGrid@2.0.4 with git auto-update

### PR DESCRIPTION
**Pull request by Amomo**
Hi @maruilian11  

For #6298, I add pqGrid at v2.0.4 and add git auto-update config in its package.json.
repo: https://github.com/paramquery/grid

Would you mind checking this PR for me? Thank you~ :-D

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : **Watch `56`, Star `316`, Fork `142`**
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

### CODE REVIEW

File: ajax/libs/pqGrid/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  "name": "pqGrid",
+  "title": "ParamQuery Grid",
+  "filename": "pqgrid.min.js",
+  "version": "2.0.4",
+  "author": {
+    "name": "Paramvir Dhindsa",
+    "email": "pdhindsa@paramquery.com",
+    "url": "http://www.paramquery.com/"
+  },
+  "licenses": "GPL-3.0",
```
Comment (PeterDaveHello) [inline ajax/libs/pqGrid/package.json]: licenses?

Comment (Amomo) [inline ajax/libs/pqGrid/package.json]: I have fix it, thank you! :-)

File: ajax/libs/pqGrid/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    "url": "https://github.com/paramquery/grid"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/paramquery/grid.git",
+    "basePath": "",
+    "files": [
+      "pqgrid*.+(js|css)",
+      "images/*.+(png|gif)"
+    ]
```
Comment (PeterDaveHello) [inline ajax/libs/pqGrid/package.json]: What about the localization files?

Comment (Amomo) [inline ajax/libs/pqGrid/package.json]: thanks for reminding me that, I almost forget!
They are added now. :-D

Comment (PeterDaveHello) [inline ajax/libs/pqGrid/package.json]: but without minified version ...

**Comment by maruilian11** [comment]
@Amomo could you please tell me why we needs files under `localize`. thanks.

**Comment by Amomo** [comment]
Because this lib provide a UI, with these `localize` files, the lib would be international for globe user.

**Comment by Amomo** [comment]
@maruilian11 I added the minified `localize/` files to "requiredFiles" field.
And, I have rebased it on latest master branch.
Would you mind to check it again? Thank you. :-D

**Comment by maruilian11** [comment]
@Amomo i think this PR is ok. could you rebase it again?

**Comment by Amomo** [comment]
@maruilian11 I have rebase this PR on latest mater branch.
Could you help me check it again? Thank you~ :-D

**Comment by maruilian11** [comment]
@Amomo sorry... i'm late..😭

**Comment by Amomo** [comment]
@maruilian11 it's okay~ Could you help me check it again? Thank you~ :-D

**Comment by maruilian11** [comment]
@PeterDaveHello i think the PR is ok.

**Comment by PeterDaveHello** [comment]
No theme? https://github.com/paramquery/grid/tree/master/themes/Office

**Comment by Amomo** [comment]
@maruilian11 I have added the theme of this lib, and rebased this PR on latest master branch.
Could you help me checking this PR again? Thank you! :-)

**Comment by maruilian11** [comment]
@PeterDaveHello i think this PR is ok.

**Comment by PeterDaveHello** [comment]
For the complex filemap, maybe you can ask author if she/he would like to create a dist folder for distribution files!

**Pull request merged**